README.md

```diff
@@ -45,12 +45,12 @@ _We want to change that!_
 
 ### 🔌 Officially supported plugins
 
-|                                                                |                                                     |                                                                                    |
-| -------------------------------------------------------------- | --------------------------------------------------- | ---------------------------------------------------------------------------------- |
-| <img width="50" src="docs/images/plugin-eslint.icon.png">      | [ESLint](./packages/plugin-eslint#readme)           | Static analysis using **ESLint** rules.                                            |
-| <img width="50" src="docs/images/plugin-coverage.icon.png">    | [Code Coverage](./packages/plugin-coverage#readme)  | Collects code **coverage** from your tests.                                        |
-| <img width="50" src="docs/images/plugin-js-packages.icon.png"> | [JS Packages](./packages/plugin-js-packages#readme) | Checks 3rd party packages for known **vulnerabilities** and **outdated** versions. |
-| <img width="50" src="docs/images/plugin-lighthouse.icon.png">  | [Lighthouse](./packages/plugin-lighthouse#readme)   | Measures web performance and best practices with **Lighthouse**.                   |
+| Plugin                                                         | Description                                                                                                                             |
```

Comment:
```suggestion
|                                                         |                                                                                                                             |
```

I left the table headers out as it was redundant info.

Reply:
I agree that the headers may seem redundant, but the whole point of this PR is to add headers as they help to reserve space for the column content and prevent shrinking. Otherwise, the table is auto-sized, and images shrink to 0 on smaller screens.

I tried out a couple of other options, but in my opinion, this one turned out to be the least awkward.

Reply:
Sry! Got it!